### ISSUE TITLE
See message information

### ISSUE THREAD

**Issue by wuniversales** [bcmapp/bcm-android#11]
It would be nice to see the information in each message.
Show the message, the date and time the message was sent and the message id.

**Comment by d0gfriend** [comment]
Yes, i already asked for the feature of '**message status**' through the BCM app but i never got an answer.
So PLEASE implement the **'transmited/received/read'-Status** feature like in Signal.
Very important.
Thanks a lot in advanced.
